src/docfx/build/xref/XrefResolver.cs

```diff
@@ -188,6 +200,28 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
             return model;
         }
 
+        // xrefs marked ValidateExternalXrefs, including uids defined internal or external repo
+        // this function return net xrefs, which defined external repo
+        private ExternalXref[] GetNetExternalXrefs()
```

Comment:
consider move this method to `JsonSchemaTransformer`.

Reply:
OK